src/aircraft_cmd.cpp

```diff
@@ -2095,7 +2095,7 @@ static bool AircraftEventHandler(Aircraft *v, int loop)
 	return true;
 }
 
-bool Aircraft::Tick()
+bool Aircraft::Ticks()
```

Comment:
Why this change? It does a single tick, so plural feels weird? Or just a mass rename? :D

Reply:
Oops, that's from a mass rename.